### DISCUSSION TITLE
How to properly align dropdown items in grids

### DISCUSSION THREAD

**Discussion by srivatsa17** [heroui-inc/heroui#2093] [Help]
I am trying to create a mega menu. Currently, I am seeing that few spaces are left at the end of each section as they are aligned column after column. Is it possible to fit another section in that vacant space if space is available for that section ? And if so, how ?

Here is my code - [CategoryMegaMenu.tsx](https://codeshare.io/78v4wk)

Sample json data - [CategoryMegaMenuData.js](https://codeshare.io/loMXmR)

What i have - 

<img width="469" alt="image" src="https://github.com/nextui-org/nextui/assets/52497355/f24b789e-4992-449d-a67c-bc63a2924727">

Expected - 

<img width="449" alt="image" src="https://github.com/nextui-org/nextui/assets/52497355/e3ccf7c6-4e79-4285-9b12-db4bf404715a">

**Comment by vitalijalbu** [comment]
Hi, I'm also looking for a **megamenu**, could you share some code pls? Thanks

**Comment by alphaxek** [comment]
Hi @srivatsa17,

Please use this `list: "grid grid-cols-2 gap-4"` instead of this `list: "grid grid-flow-col gap-4"`

Random Test:
![image](https://github.com/nextui-org/nextui/assets/116849110/945cadaa-3945-4290-b607-9bacd743954c)

**Comment by J4v4Scr1pt** [comment]
There is also a megaMenu in review: https://feedback.nextui.pro/p/megamenu

**Reply by alphaxek** [reply]
@J4v4Scr1pt thanks, I believe its a part of `NextUI Pro`
